### PR TITLE
feat: initial commit for PR reviews support

### PR DESCRIPTION
**Pull request by jpmcb**
## Description

This is the initial commit for PR reviews.

Once we have consensus on the shape of the database table, we can merge this and get the backfill to happen and then actually integrate that data for charts as part of the list feature.

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents

Related to #321 

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

File: src/pull-requests/entities/pull-request-review.entity.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import {
+  ApiModelProperty,
+  ApiModelPropertyOptional,
+} from "@nestjs/swagger/dist/decorators/api-model-property.decorator";
+import { DbUser } from "../../user/user.entity";
+import { DbPullRequest } from "./pull-request.entity";
+
+@Entity("pull_request_reviews")
+export class DbPullRequestReview extends BaseEntity {
+  @PrimaryGeneratedColumn()
```
Comment (brandonroberts) [inline src/pull-requests/entities/pull-request-review.entity.ts:11]: Should this be `PrimaryColumn`, as it's not autogenerated?

File: src/pull-requests/entities/pull-request-review.entity.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import { Entity, PrimaryGeneratedColumn, Column, ManyToOne, JoinColumn, BaseEntity, CreateDateColumn } from "typeorm";
+import {
+  ApiModelProperty,
+  ApiModelPropertyOptional,
+} from "@nestjs/swagger/dist/decorators/api-model-property.decorator";
+import { DbUser } from "../../user/user.entity";
+import { DbPullRequest } from "./pull-request.entity";
+
+@Entity("pull_request_reviews")
+export class DbPullRequestReview extends BaseEntity {
```
Comment (brandonroberts) [inline src/pull-requests/entities/pull-request-review.entity.ts:10]: We add all entities to the `imports` array in the TypeOrm section of the  `app.module.ts` also

File: src/pull-requests/pull-request.controller.ts [hunk 3]
```diff
@@ -52,4 +55,15 @@ export class PullRequestController {
       [30, 60].map(async (interval) => this.pullRequestsInsightService.getInsight(interval, pageOptionsDto))
     );
   }
+
+  @Get("/:id/reviews")
+  @ApiOperation({
+    operationId: "getPullRequestReviews",
+    summary: "",
```
Comment (brandonroberts) [inline src/pull-requests/pull-request.controller.ts:62]: Summary?

**Review by brandonroberts — CHANGES_REQUESTED** [review]

**Review by bdougie — APPROVED** [review]
🚀

**Review by OgDev-01 — APPROVED** [review]
🍕

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**